modules/consensus.go

```diff
@@ -87,6 +87,16 @@ type ConsensusSet interface {
 	// This is a thread-safe way of managing updates.
 	ConsensusSetSubscribe(ConsensusSetSubscriber)
 
+	// CurrentBlock returns the most recent block on the heaviest fork known to
+	// the consensus set.
+	CurrentBlock() types.Block
```

Comment:
This is actually only called at miner startup =/

There's probably a way to avoid this call in the miner.